smart-contract-verifier/Cargo.toml

```
@@ -14,7 +14,7 @@ chrono = "0.4"
 cron = "0.11"
 ethabi = "17.2"
 ethers-core = "0.17"
-ethers-solc = { version = "0.17", features = ["svm-solc"] }
+ethers-solc = { version = "0.17", features = ["svm-solc", "async"] }
```

```suggestion
ethers-solc = { version = "0.17", features = ["async"] }
```
Nit: actually we can remove `svm-solc` feature